couchdb_custom/couchdb_custom.py

```diff
@@ -105,3 +112,9 @@ def _get_shard_and_db(shard_name):
     """
     split = shard_name.split("/")
     return split[1], split[-1].split(".")[0]
+
+def _in_maintenance_mode(context, host):
+    response = context.request("/_up", host)
+    if response.get("status") == "maintenance_mode":
+        return True
+    return False
```

Comment:
nit
```suggestion
    return response.get("status") == "maintenance_mode"
```

also add a newline.